### PR TITLE
Psd 75/split schema into incoming and outgoing

### PR DESCRIPTION
**Pull request by iamchandonbran**
-This branch is loosely tied to PSD 75, the process to moving vuln-filter after asset-attributor
-In order to keep consistency, I split the api.yaml into 2 api specs, gateway-incoming and gateway-outgoing
-I don't think any of these changes impact PSD 75, it's mainly something I felt like should be done, to keep consistency of course

### CODE REVIEW

File: gateway-incoming.yaml [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                $ref: '#/components/schemas/AssetWithVulnerabilities'
+      x-transportd:
+        backend: app
+        enabled:
+          - "metrics"
+          - "accesslog"
+          - "requestvalidation"
+          - "responsevalidation"
+          - "lambda"
+          - "basicauth"
```
Comment (bkrebsbach) [inline gateway-incoming.yaml]: Task: the `basicauth` plugin isn't being configured here, it should probably be removed.

File: docker-compose.yaml [hunk 2]
```diff
@@ -29,12 +29,19 @@ services:
...
+  gateway-outgoing:
+    build:
+      context: .
+      dockerfile: gateway-outgoing.Dockerfile
+    ports:
+      - "8082:8082"
     environment:
       NEXPOSE_API_USERNAME:
       NEXPOSE_API_PASSWORD:
+      NEXPOSE_API_HOST:
```
Comment (bkrebsbach) [inline docker-compose.yaml]: Task: you'll need to add a few more env vars here:
* `HTTPPRODUCER_API_HOST`
* `NEXPOSE_API_TIMEOUT`

**Comment by bkrebsbach** [comment]
Nit: can you update CODEOWNERS from `@asecurityteam/secdev` to `@asecurityteam/prodsecdev`

File: docker-compose.yaml (hunk 2 above)
Comment (iamchandonbran) [reply docker-compose.yaml]: done!

**Pull request merged**